.gitignore

```diff
@@ -46,3 +46,17 @@ genrst/
 # MacOS desktop metadata files
 ######################
 .DS_Store
+
+# VSCODE
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+!.vscode/*.code-snippets
+
+# Local History for Visual Studio Code
+.history/
+
+# Built Visual Studio Code Extensions
+*.vsix
```

Comment:
please don't include these changes

Reply:
I was caught by the VSCode IDE that added these quite automatically. What IDE do you use (even if I already have 4 different ones on my machine :-) )
I will try to amend this.

Reply:
You can use whichever IDE or editor works for you, but if you need to set custom ignore paths in`repo/ .git/info/exclude`

See the section starting with "Which file to place a pattern in depends on how the pattern is meant to be used." in https://git-scm.com/docs/gitignore 

Better yet, you might be able to configure your editor not to store state in the project dir.